### ISSUE TITLE
"mystery cave chest" inaccessible?

### ISSUE THREAD

**Issue by WashedUpLinkshot** [jangler/oos-randomizer#45]
I'm admittedly not too experienced with this game so I might just be missing something, but the completion log says that "mystery cave chest" has the Satchel and is accessible with Autumn, L2 Rang, and Magnets. I can't for the life of me find anything that would be a Mystery Cave with the tools given to me (including bombing multiple walls).

**Comment by jangler** [comment]
Sorry that the names in the log file are so unhelpful; they were originally written with terseness in mind (for use in the code) but should be revised to actually be understandable. That location is the cave blocked by mushrooms on the screen north of what's normally the mystery seed tree.

**Comment by WashedUpLinkshot** [comment]
I probably would have made the connection if I had been able to get up there. Is the pathing assuming access to Mystery Tree without any way to burn down the trees on the east edge of Horon?

**Comment by jangler** [comment]
Yes, there are a few different ways to get to that area. You can burn the trees, traverse Natzu with an animal companion, swim down from Mt. Cucco, or exit Subrosia via the portal near the dance hall.

**Comment by WashedUpLinkshot** [comment]
I'm locked at South Subrosia from having no Feather, locked at North Horon from having no Bracelet, and locked at East Horon from having no Satchel. The log thinks that Magnetic Gloves is the next key in the sequence, so I'm thinking the pathing erroneously thinks you can get from South Subrosia to North with them?

**Comment by jangler** [comment]
Can you post the log file?

**Comment by WashedUpLinkshot** [comment]
[oos-randomizer_log_2018-08-20_210357.txt](https://github.com/jangler/oos-randomizer/files/2304688/oos-randomizer_log_2018-08-20_210357.txt)
